### PR TITLE
Update live view documentation to match newest go2rtc

### PR DESCRIPTION
**Pull request by felipecrs**
~I think this is no longer required since go2rtc rc 8.~

### CODE REVIEW

**Review by NickM-27 — COMMENTED** [review]
I disagree, go2rtc still recommends host networking and I think we should do the same. Especially for remote webrtc access, host networking will make that easier and is done without explicit port forwarding, it should be something in the docs.

**Comment by felipecrs** [comment]
Ok, let me park this for now. I'll do some research on the topic.

**Comment by AlexxIT** [comment]
"WebRTC works by creating a WebSocket connection on port" - this is not WebSocket connection. TCP or UDP connection.

**Comment by AlexxIT** [comment]
Also without network host can be problems with HomeKit cameras. They need mDNS port

**Comment by NickM-27** [comment]
@felipecrs the addon will automatically map both 8555 ports. I think you can update the docs to correct the errors AlexxIt pointed out. Then maybe suggest bridge mode but say host mode is recommended to try if bridge mode leads to WebRTC issues. 

Homekit camera support will require separate docs on how to map the go2rtc json file (I think it was) so the Homekit data can be kept between container restarts so we'll need to look into that later.

**Comment by felipecrs** [comment]
I updated it. Let me know what do you think.

File: docs/docs/configuration/live.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Live view options can be selected while viewing the live stream. The options are
 
 ### WebRTC extra configuration:
 
-WebRTC works by creating a WebSocket connection on port `8555`. However, it requires additional configuration:
+WebRTC works through by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
```
Comment (NickM-27) [inline docs/docs/configuration/live.md:20]: adding `through` doesn't make sense to me?

File: docs/docs/configuration/live.md [hunk 2]
```diff
@@ -17,27 +17,37 @@ Live view options can be selected while viewing the live stream. The options are
...
-
-See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
-
-```yaml
-volumes:
-  - /path/to/your/go2rtc.yaml:/config/frigate-go2rtc.yaml:ro
-```
+WebRTC works through by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
+
+* For external access, over internet, setup your router to forward port `8555` to port `8555` on the Frigate device.
```
Comment (NickM-27) [inline docs/docs/configuration/live.md:22]: We should mention the port forward should be tcp and udp

**Review by NickM-27 — CHANGES_REQUESTED** [review]
Just a couple small things

File: docs/docs/configuration/live.md (hunk 1 above)
Comment (felipecrs) [reply docs/docs/configuration/live.md:20]:
```suggestion
WebRTC works by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
```

File: docs/docs/configuration/live.md (hunk 2 above)
Comment (felipecrs) [reply docs/docs/configuration/live.md:22]:
```suggestion
* For external access, over internet, setup your router to forward port `8555` to port `8555` on the Frigate device, for both TCP and UDP.
```

**Review by NickM-27 — APPROVED** [review]

File: docs/docs/configuration/live.md [hunk 3]
```diff
@@ -17,27 +17,37 @@ Live view options can be selected while viewing the live stream. The options are
...
-  - /path/to/your/go2rtc.yaml:/config/frigate-go2rtc.yaml:ro
-```
+WebRTC works by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
+
+* For external access, over the internet, setup your router to forward port `8555` to port `8555` on the Frigate device, for both TCP and UDP.
+* For internal/local access, you will need to use a custom go2rtc config:
+    1. Create your own go2rtc config, based on [Frigate's internal go2rtc config](https://github.com/blakeblackshear/frigate/blob/dev/docker/rootfs/usr/local/go2rtc/go2rtc.yaml).
+    2. Add your internal IP to the list of `candidates`. Here is an example, assuming that `192.168.1.10` is the local IP of the device running Frigate:
+
+        ```yaml
```
Comment (felipecrs) [inline docs/docs/configuration/live.md:27]:
```suggestion
        ```yaml title="/config/frigate-go2rtc.yaml"
```

File: docs/docs/configuration/live.md [hunk 4]
```diff
@@ -17,27 +17,37 @@ Live view options can be selected while viewing the live stream. The options are
...
+
+        webrtc:
+          candidates:
+            - 192.168.1.10:8555
+            - stun:8555
+        ```
+
+    3. Mount this config file at `/config/frigate-go2rtc.yaml`. Here is an example, if you run Frigate through docker-compose:
+
+        ```yaml
```
Comment (felipecrs) [inline docs/docs/configuration/live.md:39]:
```suggestion
        ```yaml title="docker-compose.yaml"
```

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**